Presentation.Web/Controllers/API/TextController.cs

```diff
@@ -15,7 +15,7 @@ namespace Presentation.Web.Controllers.API
     [PublicApi]
     public class TextController : GenericApiController<Text, TextDTO>
     {
-        protected readonly IGenericRepository<Text> _repository;
+        private readonly IGenericRepository<Text> _repository;
```

Comment:
Hvorfor private i stedet for protected? (Nysgerrig)

Reply:
@JacobMalling protected er til properties og metoder (hvis man vil kaldes fra subklasser). private er (og iøvrigt også nedfældet i guidelines) det eneste valg til felter :-)